packages/framework/src/Resources/views/Admin/Form/complaintItems.html.twig

```diff
@@ -7,13 +7,26 @@
                     <table class="table-main table-main--with-inline-form table-fixed">
                         <thead>
                             <tr>
+                                <th class="table-col-200px">{{ 'Name'|trans }}</th>
                                 <th class="table-col-375px">{{ 'Description'|trans }}</th>
                                 <th>{{ 'Quantity'|trans }}</th>
                             </tr>
                         </thead>
                         <tbody>
                             {% for complaintItemId, complaintItemForm in form.children %}
+                                {% set orderItem = complaintItemForm.vars.value.orderItem %}
                                 <tr id="{{ complaintItemForm.vars.id }}" data-index="{{ complaintItemId }}">
+                                    <td>
+                                        {% if orderItem.product %}
```

Comment:
this will be rewritten slightly after the structure is updated in the "base" user story, right?

Reply:
rewritten in https://github.com/shopsys/shopsys/pull/3388/commits/f0380cf90017da598fa22edb6d81cc4f03dc11da

Reply:
translations dump: https://github.com/shopsys/shopsys/pull/3388/commits/4f936c07ef0bac0666b2a37f3e1a4d4c98c7e1b3